maquetado/index.html

```diff
@@ -9,6 +9,7 @@
   </head>
   <body>
     <div class="book-cover">
+      <img class="badge-image" src="image/badge.png" alt="badge Image" />
```

Comment:
Perfect, just one thing. For decoration images we don't use `img` tag. It's better to use after or before for this and set the image inside the content property. Then you can move it with position absolute. 

Remember to set position relation in book-image so it can work as an anchor for you after or before pseudo class